app/Repositories/PostRepository.php

```diff
@@ -120,7 +120,20 @@ public function create(array $data, $signupId)
     public function update($signup, $data)
     {
         if (array_key_exists('updated_at', $data)) {
-            $signup->fill(array_only($data, ['quantity', 'quantity_pending', 'why_participated', 'updated_at']));
+            // Only update if the key is set (is not null).
+            $arrayKeysToUpdate = ['why_participated', 'quantity'];
+
+            foreach ($arrayKeysToUpdate as $key => $value) {
```

Comment:
I would look into php `array_filter` to handle this type of logic (removing null values from a keyed array). Furnes did something similar in this PR https://github.com/DoSomething/phoenix/pull/7493

Reply:
ah this is much cleaner, thanks @sbsmith86 ! I updated this and think this works fine! I think this is ok but before, it only updated with `why_participated` and `quantity`. Using `array_filter`, it replaces everything that is sent through if it isn't null. I think that is ok but wanted to run by you too in case I missed something!